### PR TITLE
Add Syphillis and HepB results for Partner and place transfered out to

### PR DESCRIPTION
**Pull request by solemabrothers**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 1]
```diff
@@ -637,16 +662,22 @@
 													<label>Condition of Baby at Discharge</label>
 													<obs conceptId="99334"  answerConceptIds="99331,162953,99332,99333,165448,165449,162954" answerLabels="MSB - Macerated Still Bath,FSB - Fresh Still Birth, NND - Immediate Neonatal Death, AL - Live Baby, AL-R - Live baby referred to another unit, AL-T - Live baby transferred to another ward/ Department, BDF - Baby Born With defect" style="dropdown" id="99334"/>
 												</div>
-												<div class="col-md-6">
+												<div class="col-md-4">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: use col-md-3

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 2]
```diff
@@ -637,16 +662,22 @@
...
 													<label>Other Ward/Department</label>
 													<obs conceptId="165456" id="other-ward-or-department" />
 												</div>
-												<div class="col-md-6">
+												<div class="col-md-4">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: use col-md-3

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 3]
```diff
@@ -637,16 +662,22 @@
...
 													<label>Transferred by</label>
 													<div id="transferred-by">
 														<encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" />
 													</div>
 												</div>
+												<div class="col-md-4">
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: use use col-md-3

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 4]
```diff
@@ -637,16 +662,22 @@
 													<label>Condition of Baby at Discharge</label>
 													<obs conceptId="99334"  answerConceptIds="99331,162953,99332,99333,165448,165449,162954" answerLabels="MSB - Macerated Still Bath,FSB - Fresh Still Birth, NND - Immediate Neonatal Death, AL - Live Baby, AL-R - Live baby referred to another unit, AL-T - Live baby transferred to another ward/ Department, BDF - Baby Born With defect" style="dropdown" id="99334"/>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Add class="long-value-select"

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 5]
```diff
@@ -637,16 +662,22 @@
 													<label>Condition of Baby at Discharge</label>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml:672]: use col-md-3 for this column and for the LLIN column above

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 6]
```diff
@@ -637,16 +662,22 @@
...
 													<label>Transferred by</label>
 													<div id="transferred-by">
 														<encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" />
 													</div>
 												</div>
+												<div class="col-md-4">
+													<label>Transfered to</label>
+													<div id="transferred-to">
+														<obs conceptId="90211" id="place-transfered-to" />
+													</div>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml:689]: Have this field deactivated by default and activate it when ward transferred to is selected. Follow the same implementation as the previous two fields.

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 7]
```diff
@@ -637,16 +662,22 @@
...
 												</div>
-												<div class="col-md-6">
+												<div class="col-md-4">
 													<label>Transferred by</label>
 													<div id="transferred-by">
 														<encounterProviderAndRole encounterRole="240b26f9-dd88-4172-823d-4a8bfeb7841f" />
 													</div>
 												</div>
+												<div class="col-md-4">
+													<label>Transfered to</label>
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Typo, us transferred

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 6 above)
Comment (semujju) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml:689]: @solemabrothers this one is pending

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 5 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml:672]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 4 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 1 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 2 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 3 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml (hunk 7 above)
Comment (solemabrothers) [reply omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Done

File: omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml [hunk 8]
```diff
@@ -56,6 +57,15 @@
                     }
                 });
 
+		       //enable/disable transferred to
+		         jq("#99334").change(function () {
+					if(jq(this).find("select").find(":selected").val() == "165449") {
+					enable_fields("transferred-to");
```
Comment (semujju) [inline omod/src/main/webapp/resources/htmlforms/HMIS-MCH-006-Maternity.xml]: Indent this code

**Pull request merged**